src/tanu/Assignment20.java

```diff
@@ -0,0 +1,29 @@
+package tanu;
+
+import java.util.Scanner;
+import java.lang.Character;
+
+class Assignment20 {
+	char letter;
+	int sumcount=0;
```

Comment:
if this variable is common for other methods then we should write it at the instance level.